tests/test_mathml_woluwe/test_002.test/liblouisutdml.ini

```diff
@@ -0,0 +1,30 @@
+cellsPerLine 40
+linesPerPage 25
+interpoint no
+lineEnd \n
+pageEnd \f
+fileEnd ^z
+printPages no
+braillePages no
+numberBraillePages no
+paragraphs yes
+beginningPageNumber 1
+printPageNumberAt top
+braillePageNumberAt bottom
+printPageNumbersInContents yes
+braillePageNumbersInContents yes
+hyphenate no
+outputEncoding UTF8
+inputTextEncoding UTF8
+backFormat plain
+backLineLength 70
+formatFor textDevice
+lineFill '
+xmlheader "<?xml version='1.0' encoding='UTF8' standalone='yes'?>"
+internetAccess no
+newEntries no
+semanticFiles ../../../lbu_files/wiskunde.sem
+literarytextTable unicode.dis,../../../lbu_files/wiskunde.ctb,braille-patterns.cti
+mathtextTable     unicode.dis,../../../lbu_files/wiskunde.ctb,braille-patterns.cti
+mathexprTable     unicode.dis,../../../lbu_files/wiskunde.ctb,braille-patterns.cti
+editTable en-us-brf.dis,braille-patterns.cti,../../../lbu_files/wiskunde_edit.ctb
```

Comment:
With the existing test runner we needed only one liblouisutdml.ini which was kinda nice I think.

Reply:
Yes, it was. If I understand correctly it was then starting `file2brl` at the location where the `liblouisutdml.ini` was found. I was too lazy to replicate that behavior. And I thought it's simpler to just copy the ini file.

Is it true that the `liblouisutdml.ini` has to be in the location where you start `file2brl`?

Reply:
Yes, it needs to be in the location where you start file2brl, otherwise it will go look for it in other places, which we don't want in the tests of course. If it's easier to implement, we could also provide (generate) an empty liblouisutdml.ini file, and put the whole configuration in styles.cfg (and rename it).

Reply:
Ah, yes, I think that is easier. IIRC `styles.cfg` is already shared.